files/usr/share/cinnamon/applets/sound@cinnamon.org/applet.js

```diff
@@ -1420,6 +1420,11 @@ MyApplet.prototype = {
             this._control["change_" + type](device);
         });
 
+        let bin = new St.Bin({ x_align: St.Align.END, style_class: 'menuitem-detail' });
```

Comment:
@rjanja I'm with @pixunil on this on this one. 'popup-inactive-menu-item' is the same style used for the status label on things like switches. You can see these in the network menu. For theming purposes all of these "additional info" labels should probably share the same style class. We already have a bit of a style class mess in Cinnamon and I'm not a huge fan of adding even more if we don't need to.